src/util.ml

```diff
@@ -2101,14 +2101,14 @@ value check_xhtml s =
           } ]
 ;
 
-(* There is a hook to avoid a dependency on compilation.ml which is
+(* There is a hook to avoid a dependency on compilation.ml and commit.ml which are
    recompiled every time *)
 value compilation_time_hook = ref "";
-value compilation_time conf =
```

Comment:
OK, so there is a principle, if you don't want the user to trick you, you put the checking code on the server side :)